blockly-core/core/trashcan.js

```diff
@@ -78,6 +80,14 @@ Blockly.Trashcan.prototype.MARGIN_TOP_ = 15;
  */
 Blockly.Trashcan.prototype.MARGIN_SIDE_ = 22;
 
+
+/**
+* Extent of hotspot on all sides beyond the size of the image.
+* @type {number}
+* @private
+*/
+Blockly.Trashcan.prototype.MARGIN_HOTSPOT_ = 25;
```

Comment:
Looks like we were already doing this (so you don't need to change anything if you don't want), but it seems a little weird to be setting constants on the prototype.